src/background/Wallet/model/types.ts

```diff
@@ -45,6 +45,7 @@ interface PublicPreferences {
   recentAddresses?: string[];
   mintDnaBannerDismissed?: boolean;
   upgradeDnaBannerDismissed?: boolean;
+  lastBackupReminderHideTime?: number;
```

Comment:
Maybe more consistent with the other already created flags?
```suggestion
  backupReminderDismissedTime?: number;
```